after/ftplugin/elvish.vim

```diff
@@ -0,0 +1 @@
+setlocal commentstring=#\ %s
```

Comment:
We want to avoid ftplugins. Please try to create a PR at vim/vim ti support this natively